impl/src/main/java/org/ehcache/config/builders/CacheManagerBuilder.java

```diff
@@ -375,11 +375,14 @@ T cast(EhcacheManager ehcacheManager) {
   }
 
   /**
-   * Convenience method to get a {@link CacheManagerConfiguration} for a {@link PersistentCacheManager}.
+   * Convenience method to get a {@link CacheManagerConfiguration} for a {@link PersistentCacheManager} stored on disk. The
+   * disk resource pool should be configured as persistent as well.
```

Comment:
That is just wrong. You need to setup the `CacheManagerBuilder` so that it returns a properly configured `PersistentCacheManager` to be able to use a `disk` resource but that does not mean you want the _reload my data_ feature always.